astroquery/mast/tests/test_mast.py

```diff
@@ -177,6 +180,87 @@ def zcut_download_mockreturn(url, file_path):
     return
 
 
+###########################
+# MissionSearchClass Test #
+###########################
+
+
+def test_missions_query_region_async(patch_post):
+    responses = mast.Missions.query_region_async(regionCoords, radius=0.002)
+    assert isinstance(responses, MockResponse)
+
+
+def test_missions_query_object_async(patch_post):
+    responses = mast.Missions.query_object_async("M101", radius="0.002 deg")
+    assert isinstance(responses, MockResponse)
+
+
+def test_missions_query_object(patch_post):
+    result = mast.Missions.query_object("M101", radius=".002 deg")
+    assert isinstance(result, Table)
+
+
+def test_missions_query_region(patch_post):
+    result = mast.Missions.query_region(regionCoords, radius=0.002 * u.deg)
+    assert isinstance(result, Table)
+
+
+def test_missions_query_criteria_async(patch_post):
+    pep_id = {'sci_pep_id':'12556'}
+    obs_type = {'sci_obs_type': "SPECTRUM"}
+    instruments = {'sci_instrume': "stis,acs,wfc3,cos,fos,foc,nicmos,ghrs"}
+    datasets = {'sci_data_set_name': ""}
+    pi_lname = {'sci_pi_last_name': ""}
+    actual_duration = {'sci_actual_duration': ""}
+    spec_1234 = {'sci_spec_1234': ""}
+    release_date = {'sci_release_date': ""}
+    start_time = {'sci_start_time': ""}
+    obs_type = {'sci_obs_type': 'all'}
+    aec = {'sci_aec': 'S'}
+    responses = mast.Missions.query_criteria_async(coordinates=regionCoords,
+                                        radius=3,
+                                        conditions=[pep_id,
+                                                    obs_type,
+                                                    instruments,
+                                                    datasets,
+                                                    pi_lname,
+                                                    spec_1234,
+                                                    release_date,
+                                                    start_time,
+                                                    obs_type,
+                                                    aec])
+    assert isinstance(responses, MockResponse)
+
+
+def test_catalogs_query_criteria(patch_post):
```

Comment:
Delete this function? It seems like the same as the one above, but with the wrong name.